subjects/Elf/Msp430/a.reko/a_text.c

```diff
@@ -399,12 +399,12 @@ ui20 vTaskDelayUntil(struct Eq_n * pc, ui20 sr, Eq_n r14, Eq_n r15)
 		globals->t020A = r11_n;
 		if (r11_n >= 0x0208)
 		{
-			r14_n = &globals->t020A;
+			r14_n = (struct Eq_n *) &globals->t020A;
```

Comment:
Is this a regression?

Reply:
I'll do investigation this week.

Reply:
`0x202<p16>` has incompatible type with pointer to `globals->t020A` after previous stages of type analysis
﻿And maybe `mov.w &0202,r15` incorrectly rewritten to `r15 = 0x202<p16>`. Shouldn't be `r15 = Mem[0x202<p16>]`?

Reply:
You're right; I've checked with the MSP430 manual. The `&0202` stands for "absolute address", and should indeed be translated to `Mem[0x0202<p16>:...]`. Working on a fix now. PR can probably go ahead without waiting for my fix however.